### PR TITLE
Release 1.11.1

### PR DESCRIPTION
**Pull request by bharanidharan14**
Update setup.cfg, docs/conf.py and CHANGELOG.rst with the changes pertaining to release 1.11.1

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+1.11.1 (2022-10-27)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Fix AWS session token to be consider while AWS secrete and access key is passed via the connection UI field
```
Comment (rajaths010494) [inline CHANGELOG.rst:10]:
```suggestion
- Fix AWS session token to be considered while AWS secret and access key is passed via the connection UI field
```

Comment (bharanidharan14) [reply CHANGELOG.rst:10]: Fixed

**Review by rajaths010494 — APPROVED** [review]
LGTM

**Comment by rajaths010494** [comment]
@phanikumv do we also need databricks doc fix PR to be also included in this release if its good to be merged?

**Comment by pankajastro** [comment]
> @phanikumv do we also need databricks doc fix PR to be also included in this release if its good to be merged?

I think this has merged so we should include it

**Comment by bharanidharan14** [comment]
@phanikumv @rajaths010494 @pankajastro can you review this PR

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+1.11.1 (2022-10-28)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Fix AWS session token to be considered while AWS secret and access key passed via the connection UI field
+  (`#730 <https://github.com/astronomer/astronomer-providers/pull/730>`_)
+- Fix the return response key of SageMaker Training and Transform Operator
+  (`#737 <https://github.com/astronomer/astronomer-providers/pull/737>`_)
```
Comment (phanikumv) [inline CHANGELOG.rst:13]: These descriptions need to be more user-centric

**Review by phanikumv — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (bharanidharan14) [reply CHANGELOG.rst]: Fixed

**Review by phanikumv — CHANGES_REQUESTED** [review]
Please change the PR title and description to be inline with previous release PR's

**Comment by rajaths010494** [comment]
> Please change the PR title and description to be inline with previous release PR's

Updated the title of the PR.

**Review by phanikumv — APPROVED** [review]

**Pull request merged**